### ISSUE TITLE
Stale share and reconnection issue

### ISSUE THREAD

**Issue by s-vack** [Lolliedieb/lolMiner-releases#1566]
whenever i run the program, it says :
 dns over https resolve failed - switching to standard resolve
connected to [pool]
to many stale shares
I need some help

**Comment by jgonzis** [comment]
> whenever i run the program, it says : dns over https resolve failed - switching to standard resolve connected to [pool] to many stale shares I need some help

Could you share some photo or similar please?

**Comment by nCr78** [comment]
@jgonzis I also have extra stale shares % when mining with lolminer:

![stale](https://user-images.githubusercontent.com/1726629/169716741-e5937265-e3fb-4f82-bf5a-344044635e0d.PNG)

The first rig is using another miner as it gets too hot for dual mining (plus I was lazy to change it) and it has 0.8% stale which is normal because of my ping. The rest of the rigs are on lolminer from 1.2 to 1.44% stale. They're all in the same room, same internet, same config:

set "POOL=stratum+ssl://eth-de.flexpool.io:5555"
set "TONPOOL=https://server1.whalestonpool.com"

Should I maybe change something in my config?
I know it's not that big of a difference, but extra 0.4-0.6% stale rate is like extra miner / pool fee.

**Comment by jgonzis** [comment]
> @jgonzis I also have extra stale shares % when mining with lolminer:
> 
> ![stale](https://user-images.githubusercontent.com/1726629/169716741-e5937265-e3fb-4f82-bf5a-344044635e0d.PNG)
> 
> The first rig is using another miner as it gets too hot for dual mining (plus I was lazy to change it) and it has 0.8% stale which is normal because of my ping. The rest of the rigs are on lolminer from 1.2 to 1.44% stale. They're all in the same room, same internet, same config:
> 
> set "POOL=stratum+ssl://eth-de.flexpool.io:5555" set "TONPOOL=https://server1.whalestonpool.com"
> 
> Should I maybe change something in my config? I know it's not that big of a difference, but extra 0.4-0.6% stale rate is like extra miner / pool fee.

Which version Versión and OS are you using and which gpus?. Good think is that flexpool.io pay 100% stale. I'm with that values to same pool with wifi.

**Comment by nCr78** [comment]
> Which version Versión and OS are you using and which gpus?. Good think is that flexpool.io pay 100% stale. I'm with that values to same pool with wifi.

I'm using windows 10, latest version of lolminer and I have mixed GPUs (most RTX3070 non-lhr, some RTX3060 lhr, some RX6800), all rigs on ethernet (and fastpath from ISP)

Wait, really, they pay stale shares? I didn't know that about flexpool!

**Comment by jgonzis** [comment]
> > Which version Versión and OS are you using and which gpus?. Good think is that flexpool.io pay 100% stale. I'm with that values to same pool with wifi.
> 
> I'm using windows 10, latest version of lolminer and I have mixed GPUs (most RTX3070 non-lhr, some RTX3060 lhr, some RX6800), all rigs on ethernet (and fastpath from ISP)
> 
> Wait, really, they pay stale shares? I didn't know that about flexpool!

Yes they pay it... You have to think that a Stale could be an uncle 1.75 ETH for the pool. If you are able to do an USB with mining OS like mmpOS or HiveOS or Simplemining for a 24/48h you will see a reduction. In Linux is an implementation that bring a small extra in that Nvidia and also a reduction in the stale, in case you want to try it.
